hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/ConsumerGroupsDescriber.java

```diff
@@ -58,27 +73,46 @@ public ConsumerGroupsDescriber(KafkaNamesMapper kafkaNamesMapper, AdminClient ad
+    private ConsumerGroup getKafkaConsumerGroup(Map<TopicPartition, OffsetAndMetadata> topicPartitionOffsets,
+                                                Map<KafkaTopicName, ContentType> kafkaTopicContentTypes,
+                                                ConsumerGroupDescription description) {
         Set<ConsumerGroupMember> groupMembers = description.members().stream()
-                .map(member -> getKafkaConsumerGroupMember(topicPartitionOffsets, member)).collect(toSet());
+                .map(member -> getKafkaConsumerGroupMember(topicPartitionOffsets, kafkaTopicContentTypes, member))
+                .collect(toSet());
 
-        return new ConsumerGroup(description.groupId(), description.state().toString(), groupMembers);
+        return new ConsumerGroup(clusterName, description.groupId(), description.state().toString(), groupMembers);
     }
 
-    private ConsumerGroupMember getKafkaConsumerGroupMember(Map<org.apache.kafka.common.TopicPartition, OffsetAndMetadata> topicPartitionOffsets, MemberDescription member) {
+    private ConsumerGroupMember getKafkaConsumerGroupMember(Map<TopicPartition, OffsetAndMetadata> topicPartitionOffsets,
+                                                            Map<KafkaTopicName, ContentType> kafkaTopicContentTypes,
+                                                            MemberDescription member) {
         Set<pl.allegro.tech.hermes.api.TopicPartition> kafkaTopicPartitions = member.assignment().topicPartitions().stream().map(
                 topicPartition -> {
                     Optional<OffsetAndMetadata> offset = Optional.ofNullable(topicPartitionOffsets.get(topicPartition));
-                    return new pl.allegro.tech.hermes.api.TopicPartition(topicPartition.partition(),
+                    return new pl.allegro.tech.hermes.api.TopicPartition(
+                            topicPartition.partition(),
                             topicPartition.topic(),
                             offset.map(OffsetAndMetadata::offset).orElse(0L),
                             logEndOffsetChecker.check(topicPartition),
-                            offset.map(OffsetAndMetadata::metadata).orElse(""));
+                            offset.map(OffsetAndMetadata::metadata).orElse(""),
+                            kafkaTopicContentTypes.get(KafkaTopicName.valueOf(topicPartition.topic()))
+                    );
                 }
         ).collect(toSet());
-        return new ConsumerGroupMember(member.consumerId(), member.clientId(), member.host(), kafkaTopicPartitions);
+        return new ConsumerGroupMember(member.consumerId(), member.clientId(), toHostName(member.host()), kafkaTopicPartitions);
+    }
+
+    private static String toHostName(String inetAddressStringRepresentation) {
+        try {
+            String[] parts = inetAddressStringRepresentation.split("/");
+            String ip = parts[parts.length - 1];
```

Comment:
Looks like these 2 lines can be before try-catch block, wdyt?

Reply:
done